### ISSUE TITLE
pycodestyle allows bracket with no space after in

### ISSUE THREAD

**Issue by DRMacIver** [PyCQA/pycodestyle#646]
The following pieces of code both pass pycodestyle:

```python
s = (1, 2, 3)
for g in(s):
    print(g)
```

```python
1 in(s)
```

Note the lack of space between the in and the opening bracket.

The first is arguably unspecified, but the second is [definitely forbidden](https://www.python.org/dev/peps/pep-0008/#other-recommendations) by PEP8, so arguably the first should be too for consistency.

**Comment by jayvdb** [comment]
Ah, this is the complement of https://github.com/PyCQA/pycodestyle/issues/371 / https://github.com/PyCQA/pycodestyle/issues/570 : keyword followed by opening bracket.
There will probably be a few more similar cases with `[`, `{` and other keywords.

**Comment by peterjc** [comment]
Ah, I missed this issue when I wrote https://github.com/PyCQA/pycodestyle/issues/843#issuecomment-496212488

This issue seems to cover the E225 false positive test cases I wanted to file:

```python
"""This is an E225 test case."""
print(3 in (1, 2, 3, 4, 5))  # True; should pass QC
print(10 in(1, 2, 3, 4, 5))  # False; should fail E225?
print(None or(1, 2))  # (1, 2); should fail E225?
```

Commenting briefly again to get the keyword E225 and binary operators (in, or, and, not, etc) into this issue's text.

**Comment by asottile** [comment]
Duplicate of #371
